src/Chronicle.Integrations.EFCore/Persistence/EFCoreSagaLog.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Chronicle.Integrations.EFCore.Repositories;
+
+namespace Chronicle.Integrations.EFCore.Persistence
+{
+    internal class EFCoreSagaLog : ISagaLog
+    {
+        private readonly ISagaLogRepository SagaLogRepository;
+
+        public EFCoreSagaLog(ISagaLogRepository _sagaLogRepository)
```

Comment:
please remove underscore from all method parameters ;) Should only be applied for private fields